lib/player.js

```diff
@@ -5445,7 +5446,11 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
         return new shaka.text.UITextDisplayer(
             this.video_, this.videoContainer_);
       } else {
-        return new shaka.text.SimpleTextDisplayer(this.video_);
+        if ('addTextTrack' in this.video_) {
```

Comment:
Unsure about whether this is the cleanest and most appropriate way to do a feature check here.

Reply:
Probably also worth logging a warning pointing out that text tracks won't work, because the browser doesn't support them.

Reply:
Good call, added a line.